indexers/keyvalue/LevelDBIndexer/__init__.py

```diff
@@ -46,14 +47,39 @@ def get_query_handler(self):
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def query(self, key: str, *args, **kwargs) -> Optional['jina_pb2.Document']:
+    def query(self, key: Any) -> Optional[Any]:
```

Comment:
I think the args snd kwargs are there on purpose from a conversación I has with Han

Reply:
https://github.com/jina-ai/jina/issues/1285

Reply:
Thanks, I thought there will be no derived classes from LevelDBIndexer. I'll put the `*args, **kwargs` back.